indigo_app/templates/document/_annotations.html

```diff
@@ -2,25 +2,30 @@
 {% verbatim %}
 {{#unless permissions.readonly }}
   <div class="float-right">
-    <a href="#" class="dropdown-toggle btn btn-sm" data-toggle="dropdown"></a>
-    <div class="dropdown-menu">
-      {{#unless in_reply_to}}
-        {{#unless task}}
-          {{#if permissions.can_create_task}}
-            <a href="#" class="dropdown-item create-anntn-task">Create task</a>
+
+    {{#if id}}
```

Comment:
I don't think you should have to guard for having an id -- "fake" annotations are always replies, and so should never be top-level?

Reply:
I put the id check to disable the entire drop-down to not only eliminate `Edit` and `Delete` options, but also visually distinguish fake annotations from real ones by the lack of the dropdown as you can see on the screenshot on the PR. However, I can revert to just disabling `Edit` and `Delete` operations in the dropdown of fake annotations